iceoryx_dds/source/dds2iceoryx_app/main.cpp

```diff
@@ -49,6 +60,11 @@ int main(int argc, char* argv[])
     iox::runtime::PoshRuntime::getInstance("/gateway_dds2iceoryx");
 
     iox::dds::DDS2IceoryxGateway<> gw;
+    auto result = iox::dds::TomlGatewayConfigParser::parse();
+    if (!result.has_error())
```

Comment:
In case of error, no file-based configuration is applied. The gateway implicitly falls back to defaults and relies on service discovery to set up channels from DDS.
Service discovery is however not implemented, so I guess we should at least print a warning here.

Reply:
One point I noticed however, the gateway is prints logs explaining this already however the logs are printed by internal classes. I guess it makes more sense to leave it up to the caller to print the logs instead. I will move them here.

Reply:
:heavy_check_mark: